packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -251,7 +251,7 @@ library LibProving {
                 // doesn't have any significance.
                 ts.contestBond = tier.contestBond;
                 ts.contester = msg.sender;
-                ts.contestations += 1;
+                ts.contestations = ts.contestations + 1;
```

Comment:
I'm not the biggest fan of this because now you have to check that both values on both sides are the same. Which isn't a big thing, but the presumably 16 gas saved also isn't a big thing either hehe. In any case, fine for me. :)

Reply:
Lets revert this change, @adaki2004 and also ignore the `unchecked {++i}` one.

Reply:
Ok